strictEqual.js

```diff
@@ -0,0 +1,9 @@
+function strictEqual(variableOne, variableTwo) {
```

Comment:
Collapsing the ifs here and returning the condition instead of using `if` to return true/false is cleaner

```JavaScript
function strictEqual(variableOne, variableTwo) {
  return typeof variableOne == typeof variableTwo && variableOne == variableTwo
}
```

And since it is a one line return function you can use ES6+ implicit return to be even cleaner

```JavaScript
const strictEqual = (variableOne, variableTwo) => typeof variableOne == typeof variableTwo && variableOne == variableTwo
```